WTWSMS/history/titles/c_adana.txt

```diff
@@ -13,6 +13,14 @@
 500.1.1={
 	holder=6860
 }
+524.1.1={
+    holder=1310010
+    liege="k_cilicia"
```

Comment:
There is a problem, since the king of Cilicia only holds the titles you attributed to this character.

Reply:
Yeah,I didn't give the duke of cilicia a county so the king has at least 1

Reply:
But should the duke be king rather?

Reply:
I didn't think about it. Is the king of Cilicia historical?

Reply:
Put in the new character in-between from 534-535 otherwise.

Reply:
What do you mean? IDK even if at that year the character was alive.

Reply:
Sorry, typo, I meant 524-535.

Reply:
You mean to replace the current king of Cilicia with my character that years, don't you?

Reply:
Yes, and the other provinces held by the king. It seems like a reasonable solution to me, unless you prefer another approach, in which case we could proceed in that way.

Reply:
Nope. I'm ok with it. I'll solve it soon

Reply:
Okay, and don't hesitate to ask for help if you need it! 😉